### ISSUE TITLE
Model interface not generated in version 2.0 typescript-angular 

### ISSUE THREAD

**Issue by topce** [OpenAPITools/openapi-generator#302]
##### Description
with swagger-codegen it generated class but not with openapi-generator
with swagger-codegen it was generated 
```typescript
export interface I18nProperty { 
  [key: string]: string;
}
```

##### openapi-generator version
3.0.1

##### OpenAPI declaration file content or url

```yaml
i18nProperty:
    type: object
    additionalProperties:
      type: string
```

##### Command line used for generation

generate -i swagger.yaml -g typescript-angular -o C:\temp additional-properties ngVersion=6.0.0

##### Steps to reproduce

Add to yaml file and execute code generation for  -g typescript-angular 
```yaml
i18nProperty:
    type: object
    additionalProperties:
      type: string
```

**Comment by jmini** [comment]
I think this is a duplicate from #191 and #255

**Comment by topce** [comment]
@jmini probably

**Comment by topce** [comment]
fixed by @wing328  in branch 3.1.x 
[PR #360](https://github.com/OpenAPITools/openapi-generator/pull/360)

**Comment by topce** [comment]
welcome 
thank you for fixing bugs ;-)
